src/main/scala/Garage/DBConnection.scala

```diff
@@ -0,0 +1,17 @@
+package Garage
+
+import org.mongodb.scala.{MongoClient, MongoDatabase}
+
+object DBConnection {
```

Comment:
you should have a method here to get the client, to get the database, to get the collection, to close the connection. you're missing a few or combined them into one